dftimewolf/lib/containers/containers.py

```diff
@@ -311,23 +311,31 @@ class WorkspaceLogs(interface.AttributeContainer):
         used to generate the results.
     path (str): path to a Workspace log file.
     user_key (str): user key associated with the audit records.
+    start_time (Optional[str]): Beginning of the time period the results cover.
+    end_time (Optional[str]): End of the time period the results cover.
   """
   CONTAINER_TYPE = 'workspace_logs'
 
   def __init__(
       self, application_name: str, path: str, filter_expression: str,
-      user_key: str='') -> None:
+      user_key: Optional[str]='', start_time: Optional[str]='',
+      end_time: Optional[str]='') -> None:
     """Initializes the Workspace logs container.
 
     Args:
-      application_name (str): Name of the application the audit records are for
+      application_name (str): Name of the application the audit records are for.
+      path (str): path to a Workspace log file.
       filter_expression (str): Workspace audit logs filter expression
           used to generate the results.
-      user_key (str): user key associated with the audit records.
-      path (str): path to a Workspace log file.
+      user_key (Optional[str]): user key associated with the audit records.
+      start_time (Optional[str]): Beginning of the time period the results
+          cover.
+      end_time (Optional[str]): End of the time period the results cover.
```

Comment:
Can we specify the format of the timestamps here?

Reply:
Yes, added